lib/usdUfe/ufe/Utils.cpp

```diff
@@ -1087,7 +1087,15 @@ bool isPropertyMetadataEditAllowed(
     // Verify that the intended target layer is stronger than existing authored opinions.
     const auto strongestLayer
         = UsdUfe::getStrongerLayer(stage, targetLayer, topAuthoredLayer, true);
-    bool allowed = (strongestLayer == targetLayer);
+    /*
+     * Here there are two cases in which we want to allow the metadata edit:
+     *     1. the target layer of the edit is stronger than the strongest layer having an opinion on the matter
+     *     2. the strongest layer having an opinion on the matter is not stronger than the target layer
+     *
+     * The 2nd case appears mostly if the getStrongerLayer function fails to find which layer is stronger,
+     * returning an empty layer reference.
+    */
+    bool allowed = (strongestLayer != topAuthoredLayer);
```

Comment:
I don't understand the check.It does not even use the targeted layer. Also, it is confusing because the inequality comparison to the topAuthoredLayer seems opposite of what we would want, but is done as a "hidden" check that strongestLayer is null.

The old check must stay but other checks could be added if needed.

The correct way to write your change would be something like:

```
bool allowed = (strongestLayer == targetLayer || !strongestLayer);
```

... but even that I'm not sure we want, IIRC, the reason we didn't allow editing when the strongest layer is null was that because this indicates the layer is a non-local layer and we did not want to allow modifying references. I think this could be changed but I'll confirm with designers.